packages/xo-server/src/xapi/index.js

```diff
@@ -600,8 +600,11 @@ export default class Xapi extends XapiBase {
       throw forbiddenOperation('destroy', vm.blocked_operations.destroy.reason)
     }
 
-    if (!forceDeleteDefaultTemplate && vm.other_config.default_template === 'true') {
-      throw forbiddenOperation('destroy', 'VM is default template')
+    if (
+      !forceDeleteDefaultTemplate &&
+      (vm.other_config.default_template === 'true' || vm.is_default_template === true)
+    ) {
+      throw vmIsTemplate(vm)
```

Comment:
`forbiddenOperation` sounds more correct since it being a template isn't the issue, here.